com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -258,6 +258,15 @@ static GUIContent TempContent(string text, string tooltip, Texture i)
 
         GUIContent m_NameContent = new GUIContent();
 
+        bool IsNonBaseCameraDeferred(UniversalAdditionalCameraData additionalCameraData)
+        {
+            if (additionalCameraData.renderType == CameraRenderType.Base)
+                return false;
+
+            ScriptableRenderer renderer = additionalCameraData.scriptableRenderer;
+            return renderer is ForwardRenderer && ((ForwardRenderer)renderer).renderingMode == RenderingMode.Deferred;
```

Comment:
Suggestion to make code more readable use `as` operator:
`ForwardRenderer renderer = additionalCameraData.scriptableRenderer as ForwardRenderer;`
`return (renderer != null) ? renderer.renderingMode == RenderingMode.Deferred : false;`

Reply:
Done.